pkgs/tools/misc/nat/default.nix

```
@@ -0,0 +1,22 @@
+{ lib, rustPlatform, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "nat";
```

Stick to the binary name.  

```suggestion
  pname = "natls";
```